articles/draas/draas-sco.md

```diff
@@ -113,23 +113,23 @@ You can self-manage your protected environment via the Zerto Self-Service Portal
 
 - Perform live failovers
 
-We've disabled some areas of the Zerto Self-Service Portal as they're for services we don't presently support. These include:
+We've disabled some areas of the ZSSP as they're for services we don't presently support. These include:
```

Comment:
The Zerto Self-Service Portal (ZSSP)

Reply:
Fixed - Zerto Self-Service Portal is first mentioned at line 74 in the Frequency and Timing section. Added the acronym there and used the acronym throughout the rest of the article